components/subject/RecordAuthorStatus.tsx

```diff
@@ -12,12 +13,21 @@ import {
 } from '@heroicons/react/24/solid'
 import { differenceInDays, formatDistance } from 'date-fns'
 
+const getProfileFromRepo = (repo: ToolsOzoneModerationDefs.RepoView) => {
+  const profile = repo.relatedRecords.find(
+    (r) => r?.['$type'] === CollectionId.Profile,
+  )
+  return profile ? (profile as AppBskyActorProfile.Record) : null
+}
```

Comment:
By using the `isX` type utils, you should be able to avoid having to type cast here


```suggestion
  return repo.relatedRecords.find(AppBskyActorProfile.isRecord)
}
```

Reply:
ooh TIL!